examples/simulation/plot_simulated_raw_data_using_subject_anatomy.py

```diff
@@ -0,0 +1,181 @@
+"""
+=======================================
+Simulate raw data using subject anatomy
+=======================================
+
+This example illustrates how to generate source estimates and simulate raw data
+using subject anatomy with the :class:`mne.simulation.SourceSimulator` class.
+Once the raw data is simulated, generated source estimates are reconstructed 
+using dynamic statistical parametric mapping (dSPM) inverse operator.
+"""
+
+# Author: Ivana Kojcic <ivana.kojcic@gmail.com>
+#         Eric Larson <larson.eric.d@gmail.com>
+#         Kostiantyn Maksymenko <kostiantyn.maksymenko@gmail.com>
+#         Samuel Deslauriers-Gauthier <sam.deslauriers@gmail.com>
+
+# License: BSD (3-clause)
+
+import os.path as op 
+
+import numpy as np
+
+import mne
+from mne.datasets import sample
+
+print(__doc__)
+
+# To simulate the sample dataset, information of the sample subject needs to be
+# loaded. This step will download the data if it not already on your machine. 
+# Subjects directory is also set so it doesn't need to be given to functions.
+data_path = sample.data_path()
+subjects_dir = op.join(data_path, 'subjects')
+subject = 'sample'
+meg_path = op.join(data_path, 'MEG', subject)
+
+# First, we get an info structure from the sample subject.
+fname_info = meg_path + '/sample_audvis_raw.fif'
+info = mne.io.read_info(fname_info)
+tstep = 1 / info['sfreq']
+
+# To simulate sources, we also need a source space. It can be obtained from the
+# forward solution of the sample subject.
+fwd_fname = op.join(meg_path, 'sample_audvis-meg-eeg-oct-6-fwd.fif')
+fwd = mne.read_forward_solution(fwd_fname)
+src = fwd['src']
+
+# To simulate raw data, we need to define when the activity occurs using events
+# matrix and specify the IDs of each event.
+# Noise covariance matrix also needs to be defined.
+# Here, both are loaded from the sample dataset, but they can also be specified
+# by the user.
+
+fname_event = meg_path + '/sample_audvis_raw-eve.fif'
+fname_cov = meg_path + '/sample_audvis-cov.fif'
+
+events = mne.read_events(fname_event)
+noise_cov = mne.read_cov(fname_cov)
+
+# Standard sample event IDs. These values will correspond to the third column
+# in the events matrix.
+event_id = {'auditory/left': 1, 'auditory/right': 2, 'visual/left': 3,
+            'visual/right': 4, 'smiley': 5, 'button': 32}
+
+###############################################################################
+# In order to simulate source time courses, labels of desired active regions
+# need to be specified for each of the 4 simulation conditions.
+# Make a dictionary that maps conditions to activation strengths within
+# aparc.a2009s labels. In the aparc.a2009s parcellation:
```

Comment:
can you add the literature reference for aparc?  It gets a `[1]_` here, and then a 
```
References
----------
.. [1] Destrieux et al (2009) ... etc.
```
at the bottom, see http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html#citations